server/datastore/mysql/hosts.go

```diff
@@ -2849,42 +2849,104 @@ func (ds *Datastore) CleanupExpiredHosts(ctx context.Context) ([]uint, error) {
 	if err != nil {
 		return nil, ctxerr.Wrap(ctx, err, "getting app config")
 	}
-	if !ac.HostExpirySettings.HostExpiryEnabled {
+
+	query := `SELECT id, config from teams`
+	var teams []*fleet.Team
+	if err := sqlx.SelectContext(ctx, ds.reader(ctx), &teams, query); err != nil {
+		return nil, ctxerr.Wrap(ctx, err, "get teams")
+	}
```

Comment:
can you use `ListTeams()` here?

Reply:
I could use `ListTeams()`, but it seemed overkill for what I need, which is just the team configs. `ListTeams` is a more complex query, AND it does a 2nd DB access to get secrets. Maybe if `ListTeams` had an option not to fetch secrets, then it would be ok.